Helper/Validator.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ * Class to perform validations.
+ */
+class Validator {
+
+  /**
+   * Creating Dotenv object to access env variables.
+   */
+  function __construct() {
+    $env = new Dotenv();
+  }
+
+  /**
+   * Function to validate email
+   *
+   * @param string $email
+   *   Email id of the user.
+   * @return boolean
+   *   returns true if email is valid and false otherwise
+   */
+  function isValidEmail(string $email) {
+    if (filter_var($email, FILTER_VALIDATE_EMAIL)) {
+      $ch = curl_init(
+        "https://emailvalidation.abstractapi.com/v1/?api_key={$_ENV['API_KEY']}&email={$_POST['email']}"
+      );
+      curl_setopt($ch, CURLOPT_RETURNTRANSFER, TRUE);
+      $data = curl_exec($ch);
+      curl_close($ch);
+      $result = json_decode($data, true);
+      if (
+        ($result['is_valid_format']['value']) &&
+        ($result['is_smtp_valid']['value'])
+      ) {
+        return true;
+      }
+      return true;
```

Comment:
Shouldn't we be returning true when the email validation is succesful for both api AND FILTER_VALIDATE_EMAIL ?